File: NotificationModule/NotificationModule/Objects/Event.cs
using Google.Cloud.Location;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NotificationModule.Objects
{
    internal class Event
    {
        public string eventName { get; set; }
        public string eventType { get; set; }
        public string eventDate { get; set; }
        public string eventTime { get; set; }
        public string eventLocationName { get; set; }
        public EventLocation eventLocation { get; set; }
        public EventDetails additionalDetails { get; set; }
    }
}

File: RecommendationService/src/main/java/com/app/recommender/model/Event.java
package com.app.recommender.model;

import com.google.cloud.firestore.DocumentSnapshot;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Event {
    private String eventName;
    private String eventType;
    private String eventDate;
    private String eventTime;
    private String eventLocationName;
    private Location eventLocation;
    private EventDetails additionalDetails;

}
